Doxyfile.in

```diff
@@ -1948,7 +1948,7 @@ PREDEFINED             = LEAK_DETECTIVE \
 EXPAND_AS_DEFINED      =
 
 # If the SKIP_FUNCTION_MACROS tag is set to YES then doxygen's preprocessor will
-# remove all refrences to function-like macros that are alone on a line, have an
```

Comment:
This file is partially autogenerated, which is why it occasionally has typos in it (depends on the Doxygen version used to update the file), which is why the file is ignored in my codespell runs (the fix also makes the line too long).